packages/storybook/stories/modal-dialog.mdx

```diff
@@ -1,21 +1,49 @@
-import { ArgTypes, Meta, Stories } from '@storybook/blocks';
+import { ArgTypes, Meta, Canvas, Stories } from '@storybook/blocks';
 import { ModalDialog } from "@equisoft/design-elements-react";
 import * as ModalDialogStories from './modal-dialog.stories';
 
-<Meta of={ModalDialogStories} />
+# Modal
 
-# Modal Dialog
+## Table of contents
+    1. [Definition](#definition)
+    2. [Usage](#usage)
+    3. [Variants](#variants)
+    4. [Properties](#properties)
+    5. [Accessibility](#accessibility)
 
-The `ModalDialog` component allows you to easily set dialog modals up in your app.
+<Meta title="Components/Modal/Doc" />
 
-## Import
+## Definition
+A modal is an overlay that requires the user to interact with it and is designed to elicit a response from the user (dialog). Modals inform users about critical information and require users to make decisions. Within apps, on the web and even on mobile, modals are increasingly used to direct the user’s attention to a specific task, without taking them away from the context of the current screen [1]. 
+<Canvas of={ModalDialogStories.Default} />
 
-```js
-    import { ModalDialog, useModal } from '@equisoft/design-elements-react'
-```
+## Usage
 
-## Accessibility
+### When to use
+- The content is urgent, and users are more likely to notice it in a modal [8].
+- It is essential to collect a small amount of information before letting the user proceed further [10].
+- You have to convey important information without losing the context of the current screen [10].
+- The user is about to take an action that has serious consequences and is difficult to reverse [9].
+
+### When not to use
+- Keep modal windows concise; avoid excessive content. Lengthy modals hinder user experience, suggesting the need for a reevaluation towards simpler approaches [5].
```

Comment:
revoir la formulation étant donné que le format a changé entre notion et ici.
Une proposition:

> When content is excessive, it's best to avoid using lengthy modal windows. Opt instead for concise content to ensure a smooth user experience. Complex modals can overwhelm users, indicating the importance of exploring simpler alternatives.

Reply:
même chose pour les autres points de la liste